shared/styles/style-guide.native.js

```diff
@@ -8,34 +11,66 @@ const fontCommon = {
 }
 
 const font = {
-  fontRegular: {
-    ...fontCommon,
-    fontFamily: 'Lato',
-    fontWeight: '400'
-  },
-  fontSemibold: {
-    ...fontCommon,
-    fontFamily: 'Lato',
-    fontWeight: '600'
-  },
-  fontBold: {
-    ...fontCommon,
-    fontFamily: 'Lato',
-    fontWeight: '700'
+  [OS_IOS]: {
+    fontRegular: {
+      ...fontCommon,
+      fontFamily: 'Lato',
+      fontWeight: '400'
+    },
+    fontSemibold: {
+      ...fontCommon,
+      fontFamily: 'Lato',
+      fontWeight: '600'
+    },
+    fontBold: {
+      ...fontCommon,
+      fontFamily: 'Lato',
+      fontWeight: '700'
+    },
+    italic: {
+      fontStyle: 'italic'
+    },
+    fontTerminal: {
+      ...fontCommon,
+      fontFamily: 'Source Code Pro'
+    },
+    fontTerminalSemibold: {
+      ...fontCommon,
+      fontFamily: 'Source Code Pro',
+      fontWeight: '600'
+    }
   },
-  italic: {
-    fontStyle: 'italic'
-  },
-  fontTerminal: {
-    ...fontCommon,
-    fontFamily: 'Source Code Pro'
-  },
-  fontTerminalSemibold: {
-    ...fontCommon,
-    fontFamily: 'Source Code Pro',
-    fontWeight: '600'
+  [OS_ANDROID]: {
+    fontRegular: {
+      ...fontCommon,
+      fontFamily: 'Lato'
+    },
+    fontSemibold: {
+      ...fontCommon,
+      fontFamily: 'Lato-Semi',
+      fontWeight: 'bold'
+    },
+    fontBold: {
+      ...fontCommon,
+      fontFamily: 'Lato',
+      fontWeight: 'bold'
```

Comment:
no, it was really iffy and also the font names have to be very specific so i figured just make it a sep thing